infra/scripts/cometbft.toml

```diff
@@ -34,7 +34,7 @@ dependencies = ["cometbft-pull", "docker-network-create"]
 [tasks.cometbft-node-id]
 script = """
 id=`docker exec ${CMT_CONTAINER_NAME} cometbft show-node-id`
-ip=`curl ifconfig.me`
+ip=`./target/release/ip`
```

Comment:
Do we really need a Rust crate for this?

We spent a lot of effort now to make sure that the build and the infra scripts can work without having to compile the project. It would be nice to keep it that way.

Reply:
Fixed.